src/PublicKey/Rsa/PrivateKey.php

```diff
@@ -79,18 +79,19 @@ public function getPublicKey()
      * Encrypt using this key
      *
      * @param  string $data
+     * @param  integer $padding
      * @return string
      * @throws Exception\RuntimeException
      * @throws Exception\InvalidArgumentException
      */
-    public function encrypt($data)
+    public function encrypt($data, $padding = OPENSSL_PKCS1_PADDING)
```

Comment:
No. We cannot and will not use a default value that will result in using a known vulnerable algorithm. If we are to consider this change, the default MUST be the value introduced in 2.5.2, `OPENSSL_PKCS1_OAEP_PADDING`.